### DISCUSSION TITLE
How can I display a Vue icon here?

### DISCUSSION THREAD

**Discussion by jimnguyendev** [LazyVim/LazyVim#691] [Q&A] [answered]
I have recently started using vim, and during the process of configuring lazyVim for Vue.js, I encountered an issue where I couldn't display the Vue icon. Other things were working fine.

<img width="1433" alt="Screenshot 2023-05-01 at 19 37 32" src="https://user-images.githubusercontent.com/58972715/235452907-db04a5ae-3cf7-4c42-8e98-68a2a7340a13.png">

**Comment by dpetka2001** [comment]
Do you have your plugins updated? Just today nvim-web-devicons pushed an update about Vue icon. Maybe it has something to do with that?

**Reply by jimnguyendev** [reply]
Thank you for your answer, I have updated it to the latest version, but the icon still does not show up. I have tried restarting my computer but it still doesn't work. 😢 
<img width="977" alt="Screenshot 2023-05-01 at 22 08 55" src="https://user-images.githubusercontent.com/58972715/235474450-dd0ed01a-324b-4be3-91a1-f1fe46a0369e.png">

**Reply by dpetka2001** [reply, marked as answer]
Just a wild guess, since I'm not encountering the problem, but maybe you also have to upgrade the patched fonts that you use in your system?

**Reply by jimnguyendev** [reply]
Thank you, I have tried with the new font Caskaydia and the icon is now displayed. 🙇‍♂️
